pkg/controller/wavefrontproxy/proxyspec_internal.go

```diff
@@ -121,6 +122,22 @@ func (ip *InternalWavefrontProxy) initialize(instance *wfv1.WavefrontProxy, reqL
 
 	ip.volumeMount = make([]corev1.VolumeMount, 0, 2)
 	ip.volume = make([]corev1.Volume, 0, 2)
+	if ip.instance.Spec.Openshift {
+		ip.volumeMount = append(ip.volumeMount, corev1.VolumeMount{
+			Name:      "wavefront-proxy-storage",
```

Comment:
If there are multiple Proxy's (CR instances) created - wouldn't this try to create the same volume mount for each one of those CR instances.

Reply:
nvm, discussed with vikram, seems like this is local to pod.